### PR TITLE
Review B3 - ennosites

### PR DESCRIPTION
**Pull request by MSAbaddon**

### CODE REVIEW

File: packs/data/bestiary-effects.db/effect-praise-adachros.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "_id": "qddahkHks1iLZ4sM",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>The creature gains +1 status bonus to attack rolls, Perception checks, saving throws, and skill checks.</p>"
```
Comment (TikaelSol) [inline packs/data/bestiary-effects.db/effect-praise-adachros.json:6]:
```suggestion
            "value": "<p>The creature gains +1 status bonus to attack rolls, perception checks, saving throws, and skill checks.</p>"
```

File: packs/data/pathfinder-bestiary-3.db/ioton.json [hunk 2]
```diff
@@ -516,7 +503,28 @@
...
+                    },
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "criticalSuccess"
+                        ],
+                        "selector": "{item|_id}-damage",
+                        "text": "PF2E.PersistentDamage.Mental1d4.criticalSuccess",
+                        "title": "",
+                        "visibility": "gm"
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/ioton.json:525]:
```suggestion
                        "visibility": "gm"
                    },
                    {
                        "key": "Note",
                        "outcome": [
                            "criticalSuccess"
                        ],
                        "selector": "{item|_id}-damage",
                        "text": "PF2E.PersistentDamage.Mental1d4.criticalSuccess",
                        "visibility": "gm"
```

File: packs/data/pathfinder-bestiary-3.db/caulborn.json [hunk 3]
```diff
@@ -1713,7 +1719,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>Three or more caulborn adjacent to each other can collectively cast <em>@Compendium[pf2e.spells-srd.Scrying]{Scrying}</em> once per day (DC 25), with no limit to the duration so long as at least three caulborn remain adjacent and use no actions other than to Sustain the Spell.</p>\n<p>The daily limit applies to all caulborn who participated, meaning none of them can participate in another <em>scrying</em> that day.</p>"
+                    "value": "<p>Three or more caulborn adjacent to each other can collectively cast <em>@Compendium[pf2e.spells-srd.Scrying]{Scrying}</em> once per day (@Check[type:will|dc:25|traits:magical,divination,scrying|overrideTraits:true]), with no limit to the duration so long as at least three caulborn remain adjacent and use no actions other than to Sustain the Spell.</p>\n<p>The daily limit applies to all caulborn who participated, meaning none of them can participate in another <em>scrying</em> that day.</p>"
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-3.db/caulborn.json:1722]:
```suggestion
                    "value": "<p>Three or more caulborn adjacent to each other can collectively cast <em>@Compendium[pf2e.spells-srd.Scrying]{Scrying}</em> once per day (@Check[type:will|dc:25]), with no limit to the duration so long as at least three caulborn remain adjacent and use no actions other than to Sustain the Spell.</p>\n<p>The daily limit applies to all caulborn who participated, meaning none of them can participate in another <em>scrying</em> that day.</p>"
```

File: packs/data/pathfinder-bestiary-3.db/ioton.json [hunk 4]
```diff
@@ -101,25 +101,21 @@
             },
             "dr": [
                 {
-                    "exceptions": "except force, ghost touch, or mental; double resistance vs. non-magical",
+                    "exceptions": "except force, ghost touch, or positive; double resistance vs. non-magical",
```
Comment (SpartanCPA) [inline packs/data/pathfinder-bestiary-3.db/ioton.json:104]:
```suggestion
                    "exceptions": "except force, ghost touch, or mental; double resistance vs. non-magical",
```
image

File: packs/data/pathfinder-bestiary-3.db/ioton.json (hunk 2 above)
Comment (MSAbaddon) [reply packs/data/pathfinder-bestiary-3.db/ioton.json]: Hm, not sure I understand that change?

**Review by SpartanCPA — APPROVED** [review]

File: packs/data/pathfinder-bestiary-3.db/ioton.json (hunk 2 above)
Comment (MSAbaddon) [reply packs/data/pathfinder-bestiary-3.db/ioton.json]: Ok, finally back from camping. If I understand correctly you removed the "titel": ""?
Not sure though if this gets added by default using the implemented note rule element ui.

**Pull request merged**